src/frontends/tensorflow_common/src/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "openvino_tensorflow_common")
+get_filename_component(root_dir "${CMAKE_CURRENT_SOURCE_DIR}" DIRECTORY)
+
+file(GLOB_RECURSE LIBRARY_SRC ${root_dir}/src/*.cpp)
+file(GLOB_RECURSE LIBRARY_HEADERS ${root_dir}/include/*.hpp)
+
+add_library(${TARGET_NAME} STATIC ${LIBRARY_SRC} ${LIBRARY_HEADERS})
+target_link_libraries(${TARGET_NAME} PRIVATE openvino::util)
+add_library(openvino::frontend::tensorflow_common ALIAS ${TARGET_NAME})
+
+target_include_directories(${TARGET_NAME}
+        PUBLIC
+        $<BUILD_INTERFACE:${root_dir}/include>
+        PRIVATE
+        ${root_dir}/src
+        $<TARGET_PROPERTY:openvino::runtime::dev,INTERFACE_INCLUDE_DIRECTORIES>)
+
```

Comment:
let's try to add similar lines:
https://github.com/openvinotoolkit/openvino/blob/ffdf31fba8def51ca54d19727fced0940c22285c/src/frontends/common/CMakeLists.txt#L36-L38
to fix Azure Windows static build

Reply:
Tried with PRIVATE key word. Thank you!